Documentation/joss/notes.md

```diff
@@ -45,24 +45,71 @@ I believe we fall under the term *engineering*, offering a *library*.
 
 > Your software should be a significant contribution to the available open source software that either enables some new research challenges to be addressed or makes addressing research challenges significantly better (e.g., faster, easier, simpler).
 
-Our software enables simultaneous loading of IFC as well as PCD data while being open source - we're not aware of any other open source library supporting the same.
+Our software enables simultaneous loading of IFC (Industry Foundation Classes) as well as PCD data while being open source - we're not aware of any other open source library supporting the same.
+
+TODO: Not sure if this is still the case: OIP in unique in the way that it supported various alignment data formats used in civil engineering: LandXML, OKSTRA, IFC Alignment 4.0 and up, (InfraGML???) etc. Also the possibility to convert forth and back between those formats makes OIP unique. It also offers connections to Open Linked Data. This makes OIP a driver and enabler for open data exchange in the filled of Building Information Modelling in the context of civil engineering.
```

Comment:
This is not entirely true anymore. Those formats have been turned off for the time being, since the meta-model of OIP changed from a multitude of separate distinct models to IFC. They need to be reintroduced ...

The conversions are part of many packages available, but they are not open source - I agree that that still remains a unique feature.

Reply:
Ok - removed it